docs/helpers/Eventline.md

```diff
@@ -0,0 +1,40 @@
+# Eventline
+
+If Eventline is provided then the `value` property is mandatory.
```

Comment:
Can you provide more information on 
1. What an eventline is? 
2. When can a consumer use this?
3. How is it different than a dateline?

Reply:
sure I will add these informations